src/lib/y2journal/query_presenter.rb

```diff
@@ -96,6 +96,11 @@ def interval_description
       end
     end
 
+    # Split journalctl timestamp at "—"; localize and return the joined string back
+    def self.localize_timestamp(ts)
+      ts.map { |t| Yast::Builtins.strftime(t, "%c") }.join(" to ")
```

Comment:
This is actually a non-translatable string. No matter in which language you are, it will always be
"$DATE1 to $DATE2". The dates themselves maybe look nicer than the previous version (which uses the default journalctl output which is not exactly pretty), but we should not introduce a hardcoded English sentence.